packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -70,6 +73,7 @@ const BLOCK_EDITOR_SETTINGS = [
 	'postContentAttributes',
 	'postsPerPage',
 	'readOnly',
+	'sectionRootClientId',
```

Comment:
Does it need to be declared here if it's locked?

Reply:
It looks like it doesn't need to be declared here if it isn't part of the parent settings that we want to include.